ios/Podfile

```diff
@@ -33,14 +33,15 @@ target 'Rainbow' do
 
   use_native_modules!
   if ENV['SKIP_FLIPPER'] != "true"
-    use_flipper!()
+    # use_flipper!()
```

Comment:
I had some problems with that and I'll prefer to firstly merge this (since no one uses flipper anyway) and then try to reenable that not to keep this on hold.